src/app/public/modules/country-field/country-field.component.ts

```diff
@@ -70,6 +86,19 @@ export class SkyCountryFieldComponent implements ControlValueAccessor, OnDestroy
     return this._defaultCountry;
   }
 
+  @Input()
+  public set supportedCountries(value: string[]) {
+    this._supportedCountries = value;
+
+    if (value && value.length > 0) {
+      this.setupCountries();
+    }
+  }
+
+  public get supportedCountries(): string[] {
```

Comment:
Does the `string` represent the ISO, the name of the country, etc.? We should probably be more specific with what type of string is expected, `supportedCountryISOs` for example.